src/http-spec.ts

```diff
@@ -35,10 +35,15 @@ export interface IHttpOperation extends INode {
   request?: IHttpOperationRequest;
   responses: IHttpOperationResponse[] & { 0: IHttpOperationResponse };
   servers?: IServer[];
+  callbacks?: IHttpCallbackOperation[];
   security?: HttpSecurityScheme[][];
   deprecated?: boolean;
 }
 
+export interface IHttpCallbackOperation extends IHttpOperation {
```

Comment:
Maybe we can explicitly remove the fields we're not going to support?

```ts
export type IHttpCallbackOperation = Omit<IHttpOperation, 'servers' | 'security' | 'callbacks'> & {
  callbackName: string;
}

```

Reply:
Done 👍